components/nav/nav.html

```diff
@@ -1,20 +1,15 @@
 <nav class="nav">
-  <img
-    src="{{ component.logo.url }}"
-    alt="{{ component.logo.alt }}"
-    class="nav__logo"
-  />
   <ul class="nav__content">
+    <a href="mailto:{{ component.email }}" class="nav__button">
+      Email Now
+    </a>
     {% for link in component.links %}
     <a href="{{ link.url }}" class="nav__link">
       <li class="nav__item">{{ link.name }}</li>
     </a>
     {% endfor %}
-    <button href="mailto:{{ component.email }}" class="nav__button">
-      {{ component.email }}
-    </button>
-    <button href="tel:{{ component.phone }}" class="nav__button">
-      {{ component.phone }}
-    </button>
+    <a href="tel:{{ component.phone }}" class="nav__button">
```

Comment:
If these are buttons let's keep them <button> elements and wrap them in <a> tags as necessary.
If they're no longer buttons let's update the class name to reflect that.